### PR TITLE
Improve log message and error handling for validating admission webhook

### PR DESCRIPTION
**Pull request by dragon3**
## WHAT

This pull request improves log message and error handling for validating admission webhook.

- Normalize error message format as much as possible
- Use errors.Wrap as much as possible

This pull request focuses updating them in validating admission webhook only.
( which means this pull request doesn't care about other code for such as gcbsigner, kubernetes plugin, helm, etc )

## WHY

Currently there are some typo, log message format is not unified.
Also some functions return errors without wrapping original errors.

### CODE REVIEW

File: cmd/kritis/admission/main.go [hunk 1]
```diff
@@ -68,9 +68,7 @@ func main() {
 
 	kritisConfigs, err := kritisconfig.KritisConfigs("")
 	if err != nil {
-		errMsg := fmt.Sprintf("error getting kritis config: %v", err)
-		glog.Errorf(errMsg)
-		return
+		glog.Fatalf("failed to get kritis config: %v", err)
```
Comment (dtan4) [inline cmd/kritis/admission/main.go:71]: As well as standard `log.Fatal`, `glog.Fatal[f]` calls `os.Exit` internally.
https://godoc.org/github.com/golang/glog#Fatal

Is it okay to replace existing `glog.Errorf` to `glog.Fatalf`?

Comment (dragon3) [reply cmd/kritis/admission/main.go:71]: Yes, it's intentionally.
( The original code returns from `main` after `glog.Errorf`, which means it exits immediately. )

Comment (dtan4) [reply cmd/kritis/admission/main.go:71]: 👌

File: pkg/kritis/crd/securitypolicy/securitypolicy.go [hunk 2]
```diff
@@ -306,6 +306,7 @@ func hasRequiredAttestation(image string, attestor *Attestor, attestations []met
 					verified = true
 					break
 				}
+				glog.Warningf("failed to verify attestation signature: KeyID=%s, %v", attestation.KeyID, err)
```
Comment (dtan4) [inline pkg/kritis/crd/securitypolicy/securitypolicy.go:309]: 👍

**Review by dtan4 — APPROVED** [review]
LGTM 👍

**Comment by dragon3** [comment]
Thank you for the review!
I'm going to merge it 💪

**Pull request merged**